src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -362,7 +362,7 @@ private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] ar
             JObject json = new JObject();
             json["script"] = Convert.ToBase64String(contract.Script);
             json["state"] = engine.Execute();
-            json["gasconsumed"] = engine.GasConsumed.ToString();
+            json["gasconsumed"] = (engine.GasConsumed * NativeContract.Policy.GetExecFeeFactor(snapshot)).ToString();
```

Comment:
Maybe we can save `GetExecFeeFactor` result in another field? 
```suggestion
            json["gasconsumed"] = engine.GasConsumed.ToString();
            json["gasprice"] = NativeContract.Policy.GetExecFeeFactor(snapshot).ToString();
```

Reply:
`engine.GasConsumed` is already multiplied by the factor, isn't it?

Reply:
No, it is not multiplied by the factor.

**Smart Contract Code**
```c#
public class Contract1 : SmartContract
{
    public static bool Verify() => Feibonacci(10) == Feibonacci(8) + Feibonacci(9);

    public static bool Verify1() => Feibonacci(10) == Feibonacci(8) + Feibonacci(9);

    private static int Feibonacci(int n) => n <= 2 ? 1 : Feibonacci(n - 1) + Feibonacci(n - 2);
}
```

**invoke function**
```json
{
    "jsonrpc": "2.0",
    "method": "invokefunction",
    "params": [
        "0x31d10659088b8aa6b1f5659a6ec797df94b9a27d", "verify",
        [],
        [
            {
                "account": "0x4e7e691a701d78fd3902109ed1f814303350ca12",
                "scopes": "CalledByEntry",
                "allowedcontracts": [],
                "allowedgroups": []
            }
        ]
    ],
    "id": 1
}
```
```json
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "script": "wh8MBnZlcmlmeQwUfaK5lN+Xx26aZfWxpoqLCFkG0TFBYn1bUg==",
        "state": "HALT",
        "gasconsumed": "4925010",
        "exception": null,
        "stack": [
            {
                "type": "Boolean",
                "value": true
            }
        ],
        "tx": "AIycV0RSJksAAAAAAJj7EQAAAAAAxxYAAAESylAzMBT40Z4QAjn9eB1wGml+TgEAJcIfDAZ2ZXJpZnkMFH2iuZTfl8dummX1saaKiwhZBtExQWJ9W1IBQgxAn03ZpxADYBat1qEew/U4OwegDK9C49A4ZUvLT+kKEaNiYJtBniCR9Ah1IgeAegHxKY5g/Yd2qNQO0Ti3gK9qjCgMIQI4s5jPEi7ov9DYbIgmvng8D2UQntiF7aJh0nohX1tRQEF0dHaq"
    }
}
```

**invokecontractverify**

```json
{
    "jsonrpc": "2.0",
    "method": "invokecontractverify",
    "params": [
        "0x31d10659088b8aa6b1f5659a6ec797df94b9a27d",
        [],
        [
            {
                "account": "0x4e7e691a701d78fd3902109ed1f814303350ca12",
                "scopes": "CalledByEntry",
                "allowedcontracts": [],
                "allowedgroups": []
            }
        ]
    ],
    "id": 1
}
```

```json
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "script": "VwABeBK2JA94EZ809XgSnzTwniIDEUAaNOgYNOUZNOKes0AaNNwYNNkZNNaes0A=",
        "state": "HALT",
        "gasconsumed": "4925010",
        "exception": null,
        "stack": [
            {
                "type": "Boolean",
                "value": true
            }
        ]
    }
}
```

The gasconsumed of both is the same

Reply:
> Maybe we can save `GetExecFeeFactor` result in another field?

Is the name `gasprice` not very intuitive?
```json
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "script": "VwABeBK2JA94EZ809XgSnzTwniIDEUAaNOgYNOUZNOKes0AaNNwYNNkZNNaes0A=",
        "state": "HALT",
        "gasconsumed": "4925010",
        "gasprice": "30",
        "exception": null,
        "stack": [
            {
                "type": "Boolean",
                "value": true
            }
        ]
    }
}
```

Reply:
`GasConsumed` is accumulated through the `AddGas()` method. And the `gas` in the `AddGas()` method has been multiplied by the factor. For example:

https://github.com/neo-project/neo/blob/865de26e8081208708426ca886fbfb82890778b0/src/neo/SmartContract/ApplicationEngine.cs#L309

Reply:
I misunderstood, I thought factor was only used for verifytrigger to prevent attacks.
So the price of executing the same code through different triggers is the same.